falcon/request.py

```diff
@@ -166,6 +162,12 @@ class Request(object):
             string, the value mapped to that parameter key will be a list of
             all the values in the order seen.
 
+        form_params (dict): The mapping of form data parameter names to their
+            values. Behaves exactly as the ``params`` attribute.
+
+        raw_body (bytes): The raw POST data, if any. Note that Falcon will not
+            save the raw body unless the `save_raw_body` option is set.
```

Comment:
I think it would be helpful to explain why it defaults to not buffering/saving the body.